### DISCUSSION TITLE
subscribe/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#157] [General]
# subscribe/

🤠 Hi，我是 Meng小羽 常规订阅 Debug客栈给您提供了四种常规订阅方式，分别是 微信公众号订阅、邮箱订阅、 RSS订阅，当然还有 Github ，请根据您的需求选择性订阅：
邮箱订阅「推荐」 feed「经典」 github PS：您可以点击上方按钮，会弹出一个新的标签页供您操作，谢谢。
专栏订阅 当然，您不习惯上方方式订阅，你可以选择你感兴趣的自媒体订阅方式，目前仅支持 知乎文章专栏、腾讯云+社区、思否专栏，我还不时制作视频上传至 哔哩哔哩 网站，请根据您的需求订阅：
知乎 云+社区 思否专栏 

https://blog.debuginn.cn/subscribe/

**Comment by debuginn** [comment]
如果文章对你有帮助的话可以关注下哦～
